python/ray/data/_internal/execution/operators/output_splitter.py

```diff
@@ -233,7 +239,7 @@ def _get_location(self, bundle: RefBundle) -> Optional[NodeIdStr]:
         return bundle.get_cached_location()
 
 
-def _split(bundle: RefBundle, left_size: int) -> (RefBundle, RefBundle):
+def _split(bundle: RefBundle, left_size: int) -> Tuple[RefBundle, RefBundle]:
```

Comment:
are these typehint changes related/needed? same with `_split_meta`, `_split_block`, `_split_single_block`

Reply:
They are unrelated but the type `(RefBundle, RefBundle)` isn't a valid type so was just rolling those changes in here since I was touching output_splitter.py. Can remove if it'd be cleaner to separate.

Reply:
SG we can keep it here, thanks